modules/oc-mirror-command-reference.adoc

```diff
@@ -70,6 +70,15 @@ The following tables describe the `oc mirror` subcommands and flags:
 |`--max-per-registry <int>`
 |Specify the number of concurrent requests allowed per registry. The default is `6`.
 
+|`--oci-feature-action`
+|The action to perform when using the Technology Preview OCI feature. Options are `copy` or `mirror`.
```

Comment:
```suggestion
|The action to perform when using the Technology Preview OCI feature. The options are `copy` or `mirror`.
```
Suggestion for including an article: https://www.ibm.com/docs/en/ibm-style?topic=grammar-articles